README.md

```diff
@@ -1,11 +1,14 @@
 # TLD Mythic Rating Helper
-Mythic Rating Helper is a discord bot which can be used to check a character's mythic runs, and return to them which dungeons would improve their overall mythic rating score. This version of Mythic Rating Helper has been modified from the original version and is currently intended for use within the Discord server for the World of Warcraft guild The Last Dynasty (US - Thrall).
+Mythic Rating Helper is a Discord bot which can provide information about completing Mythic+ dungeons at certain keystone levels and how that might impact a character's Mythic+ rating. This version of Mythic Rating Helper has been modified and is currently meant to work within the Discord server for the World of Warcraft guild The Last Dynasty (US - Thrall).
```

Comment:
I'd suggest adding the server name. ie- 

> for the World of Warcraft guild The Last Dynasty (US - Thrall), **TLD Reforged**.

Reply:
Added the Discord server name as suggested. It'll need re-approval to merge since the change was made after your approval. Sorry 😅